<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Возраст:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="age_input"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Индекс массы тела:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="BMI_input"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Курение:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="smoke_cb">
           <item>
            <property name="text">
             <string>Нет</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Да</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Употребление алкоголя</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Занятия спортом:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Пол:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="sex_cb">
           <item>
            <property name="text">
             <string>Мужской</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Женский</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="alcohol_cb">
           <item>
            <property name="text">
             <string>Нет</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Да</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="sport_cb">
           <item>
            <property name="text">
             <string>Нет</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Да</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Уровень холестерина:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="cholesterol_input"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Уровень глюкзы:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="glucose_input"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Систолическое артериальное давление:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="sys_b_p_input"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Диастолическое артериальное давление:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="dis_b_p_input"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="calculate_button">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string>Результат: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
